editor.planx.uk/src/@planx/components/Feedback/Public/Public.tsx

```diff
@@ -36,22 +36,25 @@ export const PASSPORT_FEEDBACK_KEY = "_feedback";
 const FeedbackComponent = (props: PublicProps<Feedback>): FCReturn => {
   const feedbackDataSchema = createFeedbackSchema(props.feedbackRequired);
 
-  const handleSubmitFeedback = async (values: FormProps) => {
+  const logFeedback = async (values: FormProps) => {
     const metadata = await getInternalFeedbackMetadata();
     const data = {
       ...metadata,
       ...values,
       feedbackType: "component" as FeedbackView,
     };
-    const submitFeedbackResult = await insertFeedbackMutation(data).catch(
+    await insertFeedbackMutation(data).catch(
       (err) => {
         console.error(err);
       },
     );
-    props.handleSubmit?.(makeData(props, values, PASSPORT_FEEDBACK_KEY));
-    if (!submitFeedbackResult) {
-      return;
-    }
+  }
+
+  const handleSubmitFeedback = async (values: FormProps) => {
+    if (values.feedbackScore || values.userComment) await logFeedback(values);
```

Comment:
Is adding this guard clause right...? Was there a business case for always logging all feedback here that I'm forgetting (e.g. always capturing user context or other metadata)?